### PR TITLE
Pass function alias directly to enclave

### PR DESCRIPTION
**Pull request by bendecoste**
The enclave supports running functions this way directly now. There is no need to get a function id first.

### CODE REVIEW

File: cmd/cape/cmd/run.go [hunk 1]
```diff
@@ -213,76 +203,6 @@ func run(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-// TODO: This function is exported for tuner to use. Remove once tuner is using sdk.Run directly
```
Comment (justin1121) [inline cmd/cape/cmd/run.go:216]: Guessing tuner is no longer using this?

Comment (bendecoste) [reply cmd/cape/cmd/run.go:216]: Yeah, tuner is using the CLI directly.

**Review by justin1121 — DISMISSED** [review]
LGTM just having some building issues.

**Comment by bendecoste** [comment]
> LGTM just having some building issues.

Fixed. I had accidentally removed a type other files in sdk were using.

**Review by justin1121 — APPROVED** [review]

**Pull request merged**